src/templates/Homepage/index.tsx

```diff
@@ -78,7 +78,7 @@ const Homepage: React.FC = () => {
       <ic-section-container aligned="center" full-height>
         <div className="section">
           <ic-alert
-            heading="Design System and UI Kit v3.0 now in pre-release"
+            heading="Design System and UI Kit v3.0 now in stable release"
             message="Start experimenting with Dark Mode, new components and more today!"
```

Comment:
Maybe we should also change this from "experimenting" now that v3 is going to be stable? E.g.
```suggestion
            message="Start working with Dark Mode, new components and more today!"
```